src/jsp/jsp_cl.c

```diff
@@ -652,7 +465,9 @@ jsp_call_stored_procedure_ng (PARSER_CONTEXT * parser, PT_NODE * statement)
 	}
     }
 
+#if defined (CS_MODE)
```

Comment:
Why is ret_value only cleared in CS_MODE?

Reply:
In SA_MODE, `method_invoke_fold_constants()` calls `xmethod_invoke_fold_constants()` and the ret_value is deallocated at the function. To avoid double free, this macro is added.